### PR TITLE
Web-1370: Systemd, app endpoints

### PR DESCRIPTION
**Pull request by f0086**

### CODE REVIEW

File: lib/Nitrapi/Services/CloudServers/Apps/App.php [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        if ($prefix === 'set') {
+            $this->data[$key] = $args[0];
+            return $this;
+        }
+
+        throw new DataMissingException('The key ' . $key . ' does not exist. Please set first.');
+    }
+
+    public function persist() {
+        $this->_api()->dataPost($this->_url($this->getAppName() . '/update'), [
```
Comment (Kenterfie) [inline lib/Nitrapi/Services/CloudServers/Apps/App.php]: Is error handled?

File: lib/Nitrapi/Services/CloudServers/Apps/App.php [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+            'cmd' => $this->getCmd(),
+            'parameters' => $this->data
+        ]);
+        return $this;
+    }
+
+    public function install() {
+        $this->_api()->dataPut($this->_url(''), [
```
Comment (Kenterfie) [inline lib/Nitrapi/Services/CloudServers/Apps/App.php]: Error handled?

Comment (f0086) [inline lib/Nitrapi/Services/CloudServers/Apps/App.php]: No need to handle the error. The method is async and always return "success"

File: lib/Nitrapi/Services/CloudServers/Apps/App.php [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+     * - status
+     *
+     *
+     * @param string $name The method name
+     * @param array $args The args
+     * @return mixed The resulting value from $data
+     * @throws DataMissingException if the key in $data does not exist.
+     */
+    public function __call($name, $args) {
+        $method = strtolower($name[3]) . substr($name, 4); // Remove "get..."
```
Comment (f0086) [inline lib/Nitrapi/Services/CloudServers/Apps/App.php]: Thanks, I've added a check for that.

**Pull request merged**